### PR TITLE
Basic non-technical Q and A on Bittensor

### PR DESCRIPTION
**Pull request by rajkaramchedu**

### CODE REVIEW

File: docs/questions-and-answers.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+Like this, every day 7200 TAO (86,400 seconds in a day, one TAO per 12 seconds) are newly created and distributed as rewards. 
+
+:::tip See also
+See [Emissions](./emissions.md) for details, in specific the role of the root network in determining which subnet gets how much reward.
+:::
+
+### I don't want to create a subnet, can I just be a miner or a validator?
+
+Yes. But remember, you have 32 different subnets to choose from. Requirements for mining or validating are subnet-specific. Start with this [Checklist for Validating and Mining](./subnets/checklist-for-validating-mining.md). Then see the step by step instructions in [Register, Validate and Mine](./subnets/register-validate-mine.md).
```
Comment (ifrit98) [inline docs/questions-and-answers.md:66]: Maybe we can link here to the subnet repos?

Lives in the template:
https://github.com/opentensor/bittensor-subnet-template/blob/main/template/subnet_links.py

**Review by ifrit98 — APPROVED** [review]
A few small nits, but looks good!

File: docs/questions-and-answers.md (hunk 1 above)
Comment (rajkaramchedu) [reply docs/questions-and-answers.md:66]: Good point. Right below on line 70 I link to subnet pages doc. Also, in each of the above links, the subnet pages link exists.

File: docs/questions-and-answers.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+:::tip See also
+See [Introduction](./learn/introduction.md) and [Anatomy of Incentive Mechanism](./learn/anatomy-of-incentive-mechanism.md) next.
+:::
+
+
+## Mining and validation
+
+### I don't understand Bittensor mining and validation
+
+In Bittensor, the term "mining" is not related to Bitcoin mining. Similarly Bittensor "validation" has nothing to do with blockchain validation. In Bittensor, mining is "subnet mining" and validation is "subnet validation". Subnet miners perform some useful task given to them by subnet validators. See the above [General](#general) section.
```
[the remark this answers is not in the document]
Comment (garrett-opentensor) [reply docs/questions-and-answers.md:45]: I concur with philanthrope, would be great to explain blockchain validators here and contrast with subnet validators

**Pull request merged**